pcap-linux.c

```diff
@@ -2331,13 +2331,13 @@ pcap_stats_linux(pcap_t *handle, struct pcap_stat *stats)
 	long if_dropped = 0;
 
 	/*
-	 *	To fill in ps_ifdrop, we parse /proc/net/dev for the number
+	 *	To fill in ps_ifdrop, we parse /sys/class/net for the number
```

Comment:
Perhaps that should be "...we parse /sys/class/net/statistics/{if_name}/rx_{missed,fifo}_errors for the numbers", or something that separately mentions the two statistics.